### PR TITLE
Feature/3175 faq

### PR DESCRIPTION
**Pull request by sinejespersen**
#### Link to ticket

https://leantime.itkdev.dk/#/tickets/showTicket/3175

#### Description

- Faq page added

#### Screenshot of the result

image

### CODE REVIEW

File: src/components/FAQ.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+const FAQ = () => {
+  return (
+    <>
+      <h1 className="text-2xl font-extrabold">Hjælp og vejledning</h1>
+      <section className="bg-zinc-100 dark:bg-zinc-900 flex flex-row relative my-2 rounded font-bold p-5 flex flex-col mt-5">
+        Oplysninger om Aarhus Kommune Kultur og Borgerservices behandling af dine personoplysninger ved anvendelse af
+        Podwalk
+        <Link
+          to={"/personal-information-policy"}
+          className="text-emerald-400 dark:text-emerald-800 font-bold underline mt-5"
```
Comment (yepzdk) [inline src/components/FAQ.jsx:13]: Can we use `dark:text-emerald-600` for the links?
I am a bit concerned the 800 variant causes some contrast issues.

File: src/components/FAQ.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        >
+          Læs hele oplysningsteksten
+        </Link>
+      </section>
+      <section className="bg-zinc-100 dark:bg-zinc-900 flex flex-row relative my-2 rounded font-bold p-5 flex flex-col mt-5">
+        Sådan bruger du navigationen
+        <Link to={"/navigation-help"} className="text-emerald-400 dark:text-emerald-800 font-bold underline mt-5">
```
Comment (yepzdk) [inline src/components/FAQ.jsx:20]: Can we use mt-3 instead, to reduces the spacing a little?

**Review by yepzdk — COMMENTED** [review]
Good job!
I have a little adjustment regarding the space between text and link, and the link color.

File: src/components/FAQ.jsx (hunk 1 above)
Comment (sinejespersen) [reply src/components/FAQ.jsx:13]: Ofc

File: src/components/FAQ.jsx (hunk 2 above)
Comment (sinejespersen) [reply src/components/FAQ.jsx:20]: Ofc

Comment (sinejespersen) [inline src/components/FAQ.jsx:20]:
```suggestion
        <Link to={"/navigation-help"} className="text-emerald-400 dark:text-emerald-800 font-bold underline mt-3">
```

File: src/components/FAQ.jsx (hunk 1 above)
Comment (sinejespersen) [inline src/components/FAQ.jsx:13]:
```suggestion
          className="text-emerald-400 dark:text-emerald-600 font-bold underline mt-5"
```

**Review by yepzdk — APPROVED** [review]

**Pull request merged**